projects/angular/src/data/datagrid/datagrid-column-toggle.ts

```diff
@@ -31,9 +32,11 @@ import { ClrDatagridColumnToggleButton } from './datagrid-column-toggle-button';
       clrPopoverOpenCloseButton
       [attr.aria-controls]="popoverId"
       [attr.aria-owns]="popoverId"
+      [attr.aria-expanded]="openState"
       [attr.title]="commonStrings.keys.pickColumns"
     >
       <cds-icon shape="view-columns" aria-hidden="true" [attr.title]="commonStrings.keys.pickColumns"></cds-icon>
+      <span class="clr-sr-only">{{ commonStrings.keys.pickColumns }}</span>
```

Comment:
Can you provide some more information why the "title" is not enough and we need to provide the `clr-sr-only` text?

Reply:
Not sure, the issue looks to have been raised by usage of the https://www.deque.com/axe/, testing tool. 

Note, this issue pre-dates the move to replace clr-icon with cds-icon which is inert and having the sr-only text as a sibling allows the screen reader to read correct. What we have today is an inert icon but no descriptive text for the button usage.

Reply:
🤔  Maybe the issue is that we are using `title` instead of `aria-label`? I know that we use `aria-label` on the icons in `@cds/core`. It may be possible that we could replace `title` with `aria-label` and remove the `clr-sr-only` span altogether...

CC:  @scroniser in case I'm confused on anything.

Reply:
> Yes, that's correct: the `title` isn't read consistently across screen readers. It's also a violation since one can't trigger it using keyboard-only navigation.
> 
> `clr-sr-only` text is a better option :)

@scroniser

If we changed the `title` to an `aria-label` could we get rid of the `clr-sr-only` span?

Reply:
@mathisscott - there was guidance in JIRA to remove the [attr.title]'s. I have updated the PR to remove them and adjust specs accordingly.